tests/e2e/tools/lib/editor.js

```diff
@@ -64,10 +66,36 @@ async function removeAllPostTypes({editor}) {
   }
 }
 
+/**
+ * Insert new block into page using the block inserter
+ *
+ * @param {Object} page             - The page object used to interact with the page.
+ * @param {Object} params           - Additional block's parameters.
+ * @param {Object} params.blockName - The name of the block.
+ * @param {Object} params.namespace - The namespace to search if it is needed.
+ * @return {Promise<void>}          - Playwright Locator
+ */
+const searchAndInsertBlock = async (page, {blockName, namespace = ''}) => {
```

Comment:
Is going to be used like this
https://github.com/greenpeace/planet4-master-theme/pull/2419/files#diff-63f0e6680f538b7f8f083348e387fc2b82c29a4f4ff0502c69d21648ef760a46R11

Reply:
Why do we need to add this function in a separate PR? Why not in the PR you refer to here?

Reply:
Since I need to fix the specs fom [this other PR](https://github.com/greenpeace/planet4-master-theme/pull/2343).